contents/docs/error-tracking/filter-and-search-issues.mdx

```diff
@@ -0,0 +1,97 @@
+
+Keep in mind that exception events can have more than one exception in them, due to language features like exception chaining. For freeform search, we put the types, messages, functions and file paths of all exceptions into one list, and match if the token appears in any of them. For example, if you had a chained exception with the messages `MyCustomError: Failed to load user` and `Cannot read property 'age' of undefined`, searching for `cannot read name` would match the exception, because it matches *one* of the exception messages (`name` appears in the "root" one).
+
+## Filtering exception occurrences within an issue
+
+Once you've found and opened the issue you want to investigate, you can use the same search interface to filter the exception list (pictured here, below the issue details), in order to e.g. search for a particular instance of the issue that you want to investigate further. This is particularly useful in cases where, for example, some exceptions in the issue have information others don't, and you want to use that information for debugging.
+
+Here's an example from our own systems. As you can see, I've selected a recent occurrence of this issue, to try and get more context on why it's happening, but the details available don't tell me much more than the stack trace:
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/unfiltered_exception_list_8cdbf735c0.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/exception_list_unfiltered_dark_200b8be303.png"
+  alt="An issue, with an unfiltered exception list"
+  classes="rounded"
+/>
+
+Here, I've added a property filter for a property I think will be useful for my debugging, and can then easily select an occurrence that has that property set:
+
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/filtered_exception_list_7b51efa94f.png"
+  imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/exception_list_filtered_dark_70a70d9ce9.png"
+  alt="An issue, with a filtered exception list"
+  classes="rounded"
+/>
+
+## Improving search performance
+
+We try to return results to you within a second, but sometimes if you're querying over large amounts of data, it may take longer. The following can improve the search performance:
+
+- Limit the time range you're searching over - 7 days is usually enough to get a sense for the trends of an issue over time.
+- Use freeform search rather than property filters - our freeform searches are generally faster than property filters, as the total amount of data processed is smaller.
```

Comment:
```suggestion
- **Limit the time range you're searching over:** 7 days is usually enough to get a sense for the trends of an issue over time.

- **Use freeform search rather than property filters:** Our freeform searches are generally faster than property filters, as the total amount of data processed is smaller.
```